docs/dev/pint-convert.rst

```diff
@@ -77,36 +77,39 @@ With the `uncertainties` package, the experimental uncertainty in the physical
 constants is considered, and the result is given in compact notation, with the
 uncertainty in the last figures in parentheses:
 
+The uncertainty can be enabled with `-U` (by default it is not enabled):
+
 .. code-block:: console
 
-    $ pint-convert Eh eV
+    $ pint-convert -p 20 -U Eh eV
+    1 hartree = 27.211386245988471444 eV
```

Comment:
This conversion and the conversions below are a bit confusing to me. The [CODATA value seems to be `27.211386245981(30) eV`](https://en.wikipedia.org/wiki/Hartree) - it does include uncertainty (and it is also a bit different then this value, but this difference is a topic out of scope for this discussion).

Here you do use the `-U` flag, but no uncertainty is printed?

Reply:
Thanks @doronbehar !

I think I missed to update this one. Hmm, I tried this an currently this raises the same error  as yours, which is as because of [uncertainties](https://github.com/hgrecco/pint/pull/2032#discussion_r1668622802).

Reply:
I have added a fix for it